UKHomeOffice/sgmr-service
# SPCR-130 Fix linting issues

## Description
Most components with inputs had linting issues relating to a11y. 'A form label must be associated with control'. This was happening due to the linter requiring the developer to label wrap every single input.

## To Test
* Run linter on a form with input components
* You should no longer see issues relating to a11y

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
